contributing/repository/linting.md

```diff
@@ -30,7 +30,7 @@ We recommend installing the [Prettier plugin for VS Code](https://marketplace.vi
 
 You can find the format configuration in the [Prettier config](../../.prettierrc.json).
 
-## alex
+## Alex
```

Comment:
Unrelated changes
```suggestion
## alex
```

Reply:
Ahh oke, yeah I just noticed it. Would be toooo tiny for a seperate PR 😅